src/service-object.ts

```diff
@@ -106,6 +106,9 @@ export interface ResponseCallback {
 
 export type SetMetadataResponse = [r.Response];
 
+export type GetResponse =
+    [ServiceObject | null | undefined, r.Response | undefined];
```

Comment:
In this case, `ServiceObject` shouldn't be null or undefined.  Same for the response I believe.  Just `[ServiceObject, r.Response]` will do fine :)

Reply:
Indeed! I saw the callback getting called with `null` serviceObject, but now that I take a closer look that's only for the error case.